test/metabase/upload_test.clj

```diff
@@ -558,3 +562,35 @@
             (testing "Check the data was uploaded into the table correctly"
               (is (= [["Malcolm"] ["\\."] ["Han"]]
                      (rows-for-table table))))))))))
+
+(deftest mysql-settings-test
+  (testing "Ensure that local_infile is set to true for better MySQL testing"
+    (mt/test-drivers [:mysql]
+      (is (= "ON" (-> (sql-jdbc.conn/db->pooled-connection-spec (mt/db))
+                      (jdbc/query
+                       ["show global variables like 'local_infile'"])
+                      first
+                      :value))))))
+
+(deftest load-from-csv-mysql-slow-way-test
```

Comment:
not in this PR, but Soon™ I'm going to refactor out a helper to write these tests. Getting pretty repetitive.